docs/docs/glossary.md

```diff
@@ -2,15 +2,16 @@
 title: Glossary
 disableTableOfContents: true
 ---
-
+```js
```

Comment:
Please remove your code highlighting changes as they are incorrect. These are MDX components which are imported and used, not code examples :)

Reply:
alright